src/include/storage/store/string_column_chunk.h

```diff
@@ -7,12 +7,22 @@
 namespace kuzu {
 namespace storage {
 
+struct string_hash {
+    using hash_type = std::hash<std::string_view>;
+    using is_transparent = void;
```

Comment:
Yes, it's not used directly, but required for hererogenous lookup in the unordered_map (i.e. so we can search for strings using a string_view to avoid unnecessary copies).

See [the example on cppreference](https://en.cppreference.com/w/cpp/container/unordered_map/find).

Reply:
Make sense. Thanks for the pointing!